### ISSUE TITLE
Need an easy way to get the resolved URL of a request with environment variables in it

### ISSUE THREAD

**Issue by HenrikHolmIT** [a85/POSTMan-Chrome-Extension#9702]
We have a setup using environment variables in the URL for each request. These URLs are often sent to QA and Data management personel when tests fail to fix data related problems.

I need a way to easily get the resolved URL of a request that has environment variables in it.

I propose that at the end of the textbox that contains the URL there could be a little button that when clicked would copy the resolved URL into clipboard. With resolved I mean with variables replaced retrieved from the currently selected environment.

![GetResolvedUrlFromUrlTextbox](https://user-images.githubusercontent.com/46492285/110451369-6caaa480-80c4-11eb-944a-e2629e4377bb.png)

Using Postman 8.0.7 on Windows 10

**Comment by HenrikHolmIT** [comment]
Hovering over the button could show a tooltip with the resolved url.

**Comment by CSALS** [comment]
any update on this?
I wanted to submit feature request for this. 
This would be helpful

**Comment by HenrikHolmIT** [comment]
> any update on this? I wanted to submit feature request for this. This would be helpful

Not a fix per say but what I do is I activate the request then go to console and copy the resolved url from there when I need it. It works but its a little cumbersome.

A better way (but I always forget this method and just go for the console method) is to open the Documentation tab (the little page icon) on the right side of the UI. That sidebar will also hold a resolved link that you can copy.

**Comment by CSALS** [comment]
Yes currently I've been copying the URl from documentation tab. It would be good to have some shortcut to directly copy it

**Comment by HenrikHolmIT** [comment]
I agree. Sometimes you just need the resolved URL without too much fuss.
